README.md

```diff
@@ -0,0 +1,150 @@
+# NimbleHex
+
+NimbleHex allows self-hosting of Hex packages.
+
+Features:
+
+* Pluggable storage. NimbleHex ships with local filesystem and S3 implementations
+* Publishing packages via HTTP API
+* Support for multiple repositories
+* Mirroring
+
+## Setup
```

Comment:
General feedback on the docs would be very appreciated. There's some overlap between this and comments in `config/config.exs` so any ideas how to improve it would be appreciated.